docs/source/federation/hosted-gateway-definitions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hosted gateway configurations
```

Comment:
"Hosted" gateway makes me think that the gateway wouldn't be run in my infrastructure (which isn't true)! I think a better term may be `Using managed federation` or something like that. This feels like it is the start of the platform docs for using federation with the full platform yes?